src/main/java/sorald/Main.java

```diff
@@ -121,12 +140,31 @@ public void initConfig(JSAPResult arguments) {
 		this.getConfig().setPrettyPrintingStrategy(PrettyPrintingStrategy.valueOf(arguments.getString(Constants.ARG_PRETTY_PRINTING_STRATEGY)));
 		this.getConfig().setFileOutputStrategy(FileOutputStrategy.valueOf(arguments.getString(Constants.ARG_FILE_OUTPUT_STRATEGY)));
 		this.getConfig().setMaxFixesPerRule(arguments.getInt(Constants.ARG_MAX_FIXES_PER_RULE));
+		int segmentSize = arguments.getInt(Constants.ARG_MAX_FILES_PER_SEGMENT);
+		if (segmentSize <= 0) {
+			throw new RuntimeException("Segment size need to be bigger than 0");
```

Comment:
Why do you throw an exception? If one chooses the default repair mode, the size of the segment doesn't matter at all.